examples/pytorch/question-answering/trainer_qa.py

```diff
@@ -92,4 +97,9 @@ def predict(self, predict_dataset, predict_examples, ignore_keys=None):
         predictions = self.post_process_function(predict_examples, predict_dataset, output.predictions, "predict")
         metrics = self.compute_metrics(predictions)
 
+        # Prefix all keys with metric_key_prefix + '_'
+        for key in list(metrics.keys()):
+            if not key.startswith(f"{metric_key_prefix}_"):
+                metrics[f"{metric_key_prefix}_{key}"] = metrics.pop(key)
```

Comment:
This shouldn't be necessary here, instead pass along the `metric_key_prefix` to the `eval_loop` above.

Reply:
That's where I thought it should go at first too. But, it seems that metrics are disabled _before_ calling `eval_loop`.

Reply:
Ah yes, forgot that specificity of the QA Trainer, thanks for reminding me! It's good like this!